test/jdk/sun/security/x509/URICertStore/ExtensionsWithLDAP.java

```diff
@@ -136,11 +135,6 @@ public static void main(String[] args) throws Exception {
         System.setProperty("com.sun.security.enableCRLDP", "true");
```

Comment:
Same suggestion for `com.sun.security.enableCRLDP` and `com.sun.security.enableAIAcaIssuers` properties.